README.md

```diff
@@ -1 +1,289 @@
-# Sprinkler
+<div align="center">
+
+![Sprinkler logo](img/Sprinkler_light.svg#gh-light-mode-only)
+![Sprinkler logo](img/Sprinkler_dark.svg#gh-dark-mode-only)
+
+<h3><i>A non-intrusive set of extensions sprinkled over the Kotlin standard library.</i></h3>
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia/sprinkler.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.black-kamelia%22%20AND%20a:%22sprinkler%22)
+[![Build status](https://ci.black-kamelia.com/buildStatus/icon?subject=Build&job=Sprinkler%2Fmaster)](https://ci.black-kamelia.com/job/Sprinkler/)
+</div>
+
+## 📋 Summary
+
+- [📋 Summary](#-summary)
```

Comment:
Not sure about emojis for this kind of project

Reply:
fair enough